asmcli/commands/print-config.sh

```diff
@@ -1,38 +0,0 @@
-print-config_subcommand() {
-  # make sure to redirect stdout as soon as possible if we're dumping the config
-  exec 3>&1
-  exec 1>&2
-
-  ### Preparation ###
-  context_set-option "PRINT_CONFIG" 1   # used by some validations
-  parse_args "${@}"
-  validate_args
-  prepare_environment
-
-  ### Validate ###
-  validate
-
-  ### Configure ###
-  configure_package
-  post_process_istio_yamls
-
-  local USE_HUB_WIP; USE_HUB_WIP="$(context_get-option "USE_HUB_WIP")"
-  if [[ "${USE_HUB_WIP}" -eq 1 ]]; then
-    populate_fleet_info
-  fi
-  print_config >&3
-}
-
-print_config() {
-  local MANAGED; MANAGED="$(context_get-option "MANAGED")"
-
-  if [[ "${MANAGED}" -eq 1 ]]; then
-    cat "${MANAGED_MANIFEST}"
```

Comment:
for managed, it uses MANAGED_MANIFEST and do not call istioctl profile dump. Do we need to remove this(print_config) or just disable this for unmanaged?

Reply:
`MANAGED_MANIFEST` has been removed a while ago - https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/pull/1449. And provisioning managed via asmcli is on a deprecation path - 
https://cloud.google.com/service-mesh/legacy/anthos-service-mesh/managed-anthos-service-mesh/provision-managed-anthos-service-mesh-asmcli